sweep/txgenerator.go

```diff
@@ -123,6 +134,87 @@ func generateInputPartitionings(sweepableInputs []txInput,
 	return sets, nil
 }
 
+// addWalletInputs retrieves utxos from the wallet and tries adding as many as
+// required to bring the tx output value above the given minimum.
+func addWalletInputs(inputSet *txInputSet, minOutputAmt btcutil.Amount,
+	wallet Wallet) (bool, error) {
+
+	// Retrieve wallet utxos. Only consider confirmed utxos to prevent
+	// problems around RBF rules for unconfirmed inputs.
+	utxos, err := wallet.ListUnspentWitness(1, math.MaxInt32)
```

Comment:
according to the godoc minConfs can be set `=0` here? I'm not sure what is correct or if it's the same, worth checking out probably.

Reply:
Where did you see this? I find in btcwallet the following:

```
// Outputs with fewer confirmations than the minimum or more
// confs than the maximum are excluded.
```

I'd think we need to set min to 1?

Reply:
See other comment. But yeah, setting to 1 is probably correct.